pycsob/client.py

```diff
@@ -17,6 +19,150 @@ def send(self, request, **kwargs):
         return super(HTTPAdapter, self).send(request, **kwargs)
 
 
+@dataclass
+class CartItem:
+    """Cart item for creating card payment."""
+    name: str
+    quantity: int
+    amount: int
+    description: Optional[str] = None
+
+    def to_dict(self) -> OrderedDict[str, Any]:
+        """Transform to ordered dictionary."""
+        pairs = (
+            ("name", self.name[:20]),
+            ("quantity", self.quantity),
+            ("amount", self.amount),
+            ("description", self.description[:40] if self.description is not None else None),
+        )
+        return OrderedDict([(k, v) for k, v in pairs if v not in conf.EMPTY_VALUES])
+
+
+@dataclass
+class CustomerData:
+    """Customer data for creating card payment."""
+    name: str
```

Comment:
For each data definition there should be a link to the documentation in the comment.